### PR TITLE
chore: enable kurl-proxy in the admin console helm chart

### PR DESCRIPTION
**Pull request by banjoh**
This change will allow reaching admin console via a secure connection. Users get the same workflow as the KOTS addon on from kURL i.e the can upload their own certs or have self signed certs generated to secure admin console.

Related to https://app.shortcut.com/replicated/story/94724

### CODE REVIEW

File: pkg/addons/adminconsole/adminconsole.go [hunk 1]
```diff
@@ -40,12 +40,21 @@ var protectedFields = []string{
...
 }
 
+const DEFAULT_ADMIN_CONSOLE_SVC_PORT = 30000
+
 var helmValues = map[string]interface{}{
 	"minimalRBAC":   false,
 	"isHelmManaged": false,
+	// To allow older versions of Admin Console to still work,
+	// we need to retain the "admin console service". It gets
+	// removed when kurlProxy is enabled.
```
Comment (laverya) [inline pkg/addons/adminconsole/adminconsole.go:50]: There are no older versions of admin console - we pin the version of the chart + kotsadm as part of the binary, so you can remove the "handle old versions" config items

**Comment by laverya** [comment]
This should not be merged until https://github.com/replicatedhq/kots-helm/pull/32 has been merged and a new helm chart version has been tagged.

This PR should likely include a bump to the chart version, see https://github.com/replicatedhq/embedded-cluster/blob/cd4cc4c/Makefile#L7

**Comment by banjoh** [comment]
Once https://github.com/replicatedhq/kots-helm/pull/32 and https://github.com/replicatedhq/kots/pull/4344 are merged, ensure we update `ADMIN_CONSOLE_CHART_VERSION` variable in the Makefile

**Comment by xavpaice** [comment]
> Once [replicatedhq/kots-helm#32](https://github.com/replicatedhq/kots-helm/pull/32) and [replicatedhq/kots#4344](https://github.com/replicatedhq/kots/pull/4344) are merged, ensure we update `ADMIN_CONSOLE_CHART_VERSION` variable in the Makefile

Currently waiting on release for those two, plus a re-review here

**Review by laverya — APPROVED** [review]

**Pull request merged**